compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -1492,6 +1493,16 @@ void exportNode(loco::Node *node, flatbuffers::FlatBufferBuilder &builder, Seria
     ExportContext ctx{builder, md, gd};
     OperationExporter exporter{ctx};
     circle_node->accept(&exporter);
+    if (has_origins(circle_node))
+    {
+      assert(gd._operators.size() > 0);
+      const auto node_id = gd._operators.size() - 1;
```

Comment:
`_operators` isn't always increase(virtual node case). It might need to check its size before and after the visitor.